src/ts/components/ads/AdMessageLabel.ts

```diff
@@ -23,24 +44,29 @@ export class AdMessageLabel extends Label<LabelConfig> {
   configure(player: PlayerAPI, uimanager: UIInstanceManager): void {
     super.configure(player, uimanager);
 
-    const clearText = () => {
-      this.setText('');
+    let config = this.getConfig();
```

Comment:
This could be a `const`, since it's never reassigned.

```suggestion
    const config = this.getConfig();
```

same thing with most of the variables below :)

Reply:
I thought prettier catches such things now 🫠 [4dee75c](https://github.com/bitmovin/bitmovin-player-ui/pull/787/commits/4dee75ca46341a277546d927dca7bcedfe6c9cc7)

Reply:
prettier doesn't, but `eslint` should 🤔